core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-configuration/dot-experiments-configuration.component.html

```diff
@@ -43,8 +43,14 @@
 
     <div class="dot-experiment-config-content">
         <ng-container *ngIf="!vm.isLoading; else loadingTpl">
-            <p-card *ngIf="!!vm.experiment.description" data-testId="experiment-description">
-                {{ vm.experiment.description }}
+            <p-card data-testId="experiment-description">
+                <dot-experiments-inplace-edit-text
+                    [disabled]="vm.experimentStatus === experimentStatus.RUNNING"
+                    [emptyText]="'experiments.configure.description.add'"
```

Comment:
maybe `emptyTextLabel` ?

Reply:
It is more like a `<p>`,  not a `placeholder` for the `input`, or not a `label` of the `input`. It's more like an empty text message, I will use emptyTextMessage.